README.md

```diff
@@ -41,7 +50,11 @@ npm install react-redux
 ```
 - Add React Router [React Router](https://v5.reactrouter.com/web/guides/quick-start)
 
-
+## How to run the test
+Clone the project: ```https://github.com/hamid-murambiwa/React-Capstone.git```
+Skip if you have Node installed. Otherwise, download [nodejs](https://nodejs.org/en/download/) install it on your system
+Install the node modules: ```npm install```
+Run the app: ```npm test``` or ```yarn test```
 
 ## Authors
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, please kindly remove the `s` in `Authors` since you worked on this project solo.